.github/workflows/project-keeper-verify.yml

```diff
@@ -0,0 +1,55 @@
+name: Project Keeper Verify
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  project-keeper-verify:
+    runs-on: ubuntu-latest
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+      cancel-in-progress: true
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: 11
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
```

Comment:
Is go `1.17` needed or should we use directly `1.18` or `1.19`?

Reply:
You are right. But this file is generated by project-keeper. Once PK is updated, this will automatically fix this.